com.avaloq.tools.ddk.xtext.expression/src/com/avaloq/tools/ddk/xtext/expression/generator/GenModelUtil2.java

```diff
@@ -121,7 +119,7 @@ public static GenModel findGenModel(final EModelElement eModelElement, final Res
     }
 
     try {
-      GenPackage genPackage = GenModelAccess.getGenPackage(ePackage, resourceSet);
+      GenPackage genPackage = org.eclipse.xtext.xtext.generator.util.GenModelUtil2.getGenPackage(ePackage, resourceSet);

```

Comment:
we should use import as usual

Reply:
I am using the qualified name because this class is referenced under a class of the same name.
Do you want me to rename GenModelUtil3 here? I am not sure if it is safe to do so.